codegen/src/rhai_module.rs

```diff
@@ -6,7 +6,7 @@ use crate::attrs::ExportScope;
 use crate::function::ExportedFn;
 use crate::module::Module;
 
-pub(crate) type ExportedConst = (String, syn::Expr);
+pub(crate) type ExportedConst = (String, Box<syn::Type>, syn::Expr);
```

Comment:
Why is this a `Box` instead of just a `syn::Type`? If it just for the `clone`, then see my code suggestion earlier.

Reply:
No particular reason, except that I copied it from some other struct...  Not very familiar with `syn::Type` to know the difference.

Reply:
I'll fix it.